src/services/shopify-client.ts

```diff
@@ -0,0 +1,25 @@
+export enum SubscriberListState {
+  ready,
+  synchronizing_contacts,
+}
+
+export interface SubscriberList {
+  name: string;
+  id: number;
+  amount_subscribers: number;
```

Comment:
What do you think about:

```suggestion
  subscribersCount: number;
```